services/lgtm/lgtm-alerts.service.js

```diff
@@ -35,7 +41,8 @@ module.exports = class LgtmAlerts extends LgtmBaseService {
 
   static render({ alerts }) {
     return {
-      message: metric(alerts) + (alerts === 1 ? ' alert' : ' alerts'),
+      label: 'alerts',
```

Comment:
FYI it's not necessary to set a static label value when you've already done so in `defaultBadgeData`. In some cases the label value is determined dynamically based on input/fetch response/etc. and so it's necessary to be able to set the `label` as part of rendering. However, in most cases this is a static value and handled via the default badge data :+1:

Reply:
Got it!